trident/spectrum_generator.py

```diff
@@ -598,11 +604,15 @@ def add_gaussian_noise(self, snr, seed=None):
         >>> sg.add_gaussian_noise(10)
         >>> sg.plot_spectrum('spec_noise.png')
         """
+        self.snr = snr
         np.random.seed(seed)
         noise = np.random.normal(loc=0.0, scale=1/float(snr),
```

Comment:
Separate issue, but this should be `1 / np.qsrt(snr)` since the `scale` keyword is the standard deviation.  You can prove this to your self with:
```
x = np.random.normal(scale=10, size=10000)
print (np.std(x))
```

Reply:
Similarly, for the `add_noise_vector` function below, you can set `self.SNR = 1 / np.std(noise)**2`.

Reply:
I'm confused.  You are correct in that the `scale` kwarg is the standard deviation.  But the relationship between SNR and standard deviation is: SNR = mu / sigma .  So I think what we have in the code is correct, and I'm not sure why we need an additional np.sqrt(snr) needed.  It seems like your example only proves this, no?  Are you thinking of the variance and not the standard deviation?  I remain confused by your comment.

Reply:
The issue is that SNR goes as 1 / variance, not 1 / sigma, and `variance = sigma^2`.

Reply:
Just to finalize things here, we have agreed that it should be 1 / sigma after all.

Reply:
And just to elaborate, the main reason for us keeping this is that SNR appears to be related to sigma**-1 for intensity measures (like our spectra), and SNR is related to sigma**-2 for power measures.  So we're keeping it as is for now.